### PR TITLE
Disabled Username in Update-Profile.html

### PR DESCRIPTION
**Pull request by Shashi456**

### CODE REVIEW

**Comment by Shashi456** [comment]
@deshraj  can you take a look at this please?

File: frontend/src/views/web/update-profile.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
                 <div class="ev-md-container ev-card-body update-profile-card">
                     <form name="updateProfileForm" ng-submit="updateProfile.updateProfile(updateProfileForm.$valid)" novalidate>
                         <div class="input-field align-left">
-                            <input id="username" name="username" type="text" class="text-dark-black dark-autofill w-400" ng-model="updateProfile.user.username" ng-maxlength="30" ng-minlength="3" required focus-if="updateProfile.user.username">
+                            <input id="username" name="username" type="text" class="text-dark-black dark-autofill w-400" ng-model="updateProfile.user.username" ng-maxlength="30" ng-minlength="3" required focus-if="updateProfile.user.username" readonly="readonly">
```
Comment (spyshiv) [inline frontend/src/views/web/update-profile.html]: @Shashi456 : Please use `disabled` attribute instead of `readonly`.

**Comment by Shashi456** [comment]
@spyshiv  can you take a look at it now?

**Comment by spyshiv** [comment]
@Shashi456 : LGTM . Merging this.

**Comment by Shashi456** [comment]
Thanks a lot , can you point me to any other issues i can look at ?

**Comment by spyshiv** [comment]
@Shashi456 : You can start with small issues like [this](https://github.com/Cloud-CV/EvalAI/issues/811)
Comment there if you want to work on this.

**Pull request merged**